sdk/mixedreality/azure-mixedreality-authentication/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## Version 1.0.0-beta.1 (2021-01-18)
+
+- Initial release.
```

Comment:
I'm not sure how to do that without checking in broken links. It's kind of a chicken and egg thing.

Reply:
I'm happy to update this after they're checked in, but I thought the link checker would be upset about it since you can't use relative links and we'd link to the master branch.

Reply:
No worries, this can be updated in a follow-up PR.

Reply:
Apparently the changelog doesn't get checked. I've borrowed some language from the communication library.